Cargo.toml

```diff
@@ -14,7 +14,7 @@ Random number generators and other randomness functionality.
 keywords = ["random"]
 
 [dependencies]
-log = "0.2.1"
+log = "*"
```

Comment:
Yes wildcard dependencies are generally an antipattern, can this be `0.3.0` instead?